tests/infra/notification.py

```diff
@@ -64,9 +65,8 @@ def notification_server(server_info, checker=None):
     else:
         raise ValueError("Notification server host:port configuration is invalid")
 
-    ThreadedTCPServer.allow_reuse_address = True
```

Comment:
I removed this line early, before realising how much of the TCPServer and HTTPServer logic were shared. @jumaffre: Do you know if it should still be present?

Reply:
Since the IP/Port of the notification server are randomly allocated, let's remove it.